cvat-ui/src/components/create-cloud-storage-page/cloud-storage-form.tsx

```diff
@@ -195,7 +195,6 @@ export default function CreateCloudStorageForm(props: Props): JSX.Element {
 
         if (cloudStorage) {
             cloudStorageData.id = cloudStorage.id;
-            // TODO: it may be coincide with the real account name
```

Comment:
This was my delusion. I checked the documentation and the fake data cannot coincide with the real ones because in real ones they can only be used lowercase letters